tests/snmp/test_snmp_queue.py

```diff
@@ -11,14 +10,20 @@
 def test_snmp_queues(duthosts, enum_rand_one_per_hwsku_hostname, localhost, creds_all_duts,
                      collect_techsupport_all_duts):
     duthost = duthosts[enum_rand_one_per_hwsku_hostname]
-    if duthost.is_supervisor_node():
-        pytest.skip("interfaces not present on supervisor node")
+    q_keys = []
+
     hostip = duthost.host.options['inventory_manager'].get_host(
         duthost.hostname).vars['ansible_host']
 
-    q_keys = redis_get_keys(duthost, "CONFIG_DB", "QUEUE|*")
+    for asic_id in duthost.get_asic_ids():
+        namespace = duthost.get_namespace_from_asic_id(asic_id)
+        sonic_db_cmd = "sonic-db-cli {}".format("-n " + namespace if namespace else "")
```

Comment:
`SonicAsic` class already  function to run sonic-db-cli on a given asic. Use it directly?
https://github.com/sonic-net/sonic-mgmt/blob/21784aea431323bf4516d640342f94c3fe8070d5/tests/common/devices/sonic_asic.py#L156

Reply:
Modified as per comment